dependencies/ACE/CMakeLists.txt

```diff
@@ -97,94 +101,103 @@ else()
   set ( _INSTALL_COMMAND ${_MAKE} install )
 endif()
 
-ExternalProject_Add( ${${PROJECT_NAME}_EXTERN}
-   URL ${${PROJECT_NAME}_URL}
-   URL_HASH SHA256=${${PROJECT_NAME}_SHA265}
+string(REPLACE "." "_" _VERSION_UNDERSCORE ${PROJECT_VERSION})
+
+set( EXTERN ${PROJECT_NAME}-extern )
+include(ExternalProject)
+ExternalProject_Add( ${EXTERN}
+   URL "https://github.com/DOCGroup/ACE_TAO/releases/download/ACE%2BTAO-${_VERSION_UNDERSCORE}/ACE-${PROJECT_VERSION}.tar.gz"
+   URL_HASH SHA256=${SHA265}
```

Comment:
What the what??? I don't get it.  Is the typo intentional because of a name conflict or something?  Is the variable used anywhere else?  If not, can we just use the raw hash here and get rid of the poorly-named variable altogether?

Reply:
Huh? It’s a 256bit SHA, what name would you like?